src/release/issue-triaging.md

```diff
@@ -1,125 +1,276 @@
+    - `regression-untriaged`: unclear what kind of regression and its nature,
+      indicates further regression triaging is necessary.
+    - `regression-from-stable-to-stable`: something that regressed from an older
+      stable release, which has already reached the latest stable release or
+      even earlier stable releases.
+    - `regression-from-stable-to-{beta,nightly}`: something that regressed from
+      the a stable release to beta or nightly channel.
+- [`I-*`]: Different labels about the nature[^2] of a bug.
+    - `I-ICE`: Internal Compiler Error.
+    - `I-prioritize`: indicates that the issue should be additionally triaged by
+      the [T-compiler prioritization working group
+      (WG-priority)](../compiler/prioritization.md) to assign a priority `P-*`
+      if applicable to indicate the urgency.
+    - `I-heavy`: heavy code (binary size).
+    - `I-slow`: slow run-time performance.
+    - `I-hang`: the compilation fails to complete after a long time.
+    - `I-crash`: the compiler or generated code crashes but does not manifest as
+      an ICE, e.g. SIGSEGV or access violations.
+    - `I-unsound`: library, compiler, type system unsoundness.
+    - `I-compilemem`: excessive memory usage during compilation.
+    - `I-compiletime`: slow compilation time. 
+    - `I-{team}-nominated`: issue is nominated for discussion by `{team}`. E.g.
+      `I-compiler-nominated`.
+- [`P-*`]: Priority labels. Applied using the [compiler prioritization
+  procedure](../compiler/prioritization.md).
+- [`E-*`]: Calls for participation[^3], for example to minimize an issue.
+    - `E-mentor`: A mentor is available to help with the issue, which makes for
+      good first issues. Make sure that the intended mentor is actually willing
+      and available to mentor.
+    - `E-needs-mcve`: This issue has a reproduction, but it is not minimal, it
+      should be minimized.
+    - `E-needs-bisection`: This issue needs a bisection, for example using
+      [cargo-bisect-rustc](https://github.com/rust-lang/cargo-bisect-rustc).
+    - `E-needs-investigation`: This issue needs further investigation to
+      determine root causes and the nature of the issue.
+    - `E-needs-design`: This issue will require some substantial design effort
+      (exploration, prototyping, discussions, etc.).
+    - `E-needs-test`: The issue has been fixed, but no test has been added for
+      it. After someone adds a test, it can be closed.
+    - `E-{easy,medium,hard}`: Someone has estimated how hard the issue is to
+      fix. This can help with finding good first issues, but is [bound to be
+      inaccurate](https://en.wikipedia.org/wiki/Curse_of_knowledge).
+- [`S-*`]: The status of an issue, for example `S-needs-repro` or
+  `S-needs-info`.
+    - `S-needs-info`: needs more information from issue reporter.
+    - `S-needs-repro`: needs reproducer (code sample, repro steps, etc.).
+    - `S-bug-has-test`: issue has a known-bug test in [`rust-lang/rust`]
```

Comment:
Is the term *known-bug* explained/defined anywhere in the forge already? If not, we might want to rephrase *known-bug test* as *test marked with compiletest directive `//@ known-bug`* or something similar.

Reply:
So I didn't actually spell out the directive, because some known-bugs are actually just normal this-doesnt-compile `//@xxx-fail` tests, if you know what I mean